src/nvim/lua/treesitter.c

```diff
@@ -144,18 +146,27 @@ int tslua_has_language(lua_State *L)
   return 1;
 }
 
+// Creates the language into the internal language map.
+//
+// Returns true if the language is correctly loaded in the language map
 int tslua_add_language(lua_State *L)
 {
   const char *path = luaL_checkstring(L, 1);
   const char *lang_name = luaL_checkstring(L, 2);
+  const char *internal_lname = lang_name;
```

Comment:
isn't this reversed? seems like `lang_name` is used for the internal map name, while `internal_lname` is the external symbol..

Reply:
Yeah maybe `symbol_name` is a better choice here.